### PR TITLE
Adds ashriek and 2 new vox plants

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About the Pull Request

Port from Baystation12/Baystation12/pull/31055

Adds *ashriek and normal *shriek to the Vox Armalis.

Adds 2 new plants that spawn on their ship.

## Why It's Good For The Game

*Vox content.*

## Did you test it?

Yes, it's fancy and all that. Doesn't runtime.

## Changelog

:cl: Imienny, RustingWithYou
rscadd: Adds new emote to Vox Armalis *ashriek, *shriek but cooler.
tweak: The *ashriek emote sound may also play when speaking.
rscadd: Adds two new Vox plants - the resin plant and the breather. The breather produces nitrogen gas, and the resin produces a new material, Vox bioresin. Adds seeds for both of them to the Vox offship.
rscadd: Adds seed extractor to Vox ship.
rscadd: Vox Armalis can now use *shriek emote.
imageadd: Adds sprites for Vox Armalis wearing the two types of Vox clothing.
/:cl:

### CODE REVIEW

File: code/modules/emotes/definitions/vox.dm [hunk 1]
```diff
@@ -1,4 +1,9 @@
 /decl/emote/audible/vox_shriek
-	key ="shriek"
+	key = "shriek"
 	emote_message_3p = "USER SHRIEKS!"
 	emote_sound = 'sound/voice/shriek1.ogg'
+
+/decl/emote/audible/armalis_shriek
+	key ="ashriek"
```
Comment (Cupax3) [inline code/modules/emotes/definitions/vox.dm]: Why was this adjusted in one placed but not the other?

Comment (EgorDinamit) [reply code/modules/emotes/definitions/vox.dm]: Good question. Will adjust both.

File: code/modules/species/outsider/vox.dm [hunk 2]
```diff
@@ -151,6 +151,9 @@
 	damage_mask =     'icons/mob/human_races/species/vox/damage_mask_armalis.dmi'
 	blood_mask =      'icons/mob/human_races/species/vox/blood_mask_armalis.dmi'
 
+	speech_sounds = list('sound/voice/ashriek.ogg')
+	speech_chance = 10
+
```
Comment (Shadowtail117) [inline code/modules/species/outsider/vox.dm:156]: Might want to put this in the changelog just in case someone cares about it, other than that LGTM

Comment (EgorDinamit) [reply code/modules/species/outsider/vox.dm:156]: It is same with normal Vox, the only difference is that Armalis use ashriek instead.
But sure, can add into changelog.

**Review by Shadowtail117 — APPROVED** [review]

**Pull request merged**